bot/decorators.py

```diff
@@ -1,9 +1,9 @@
 import logging
 import random
-import typing
 from asyncio import Lock, sleep
 from contextlib import suppress
 from functools import wraps
+from typing import Callable, Container, Union
```

Comment:
Suggestion for if you agree with the suggestion for the `inner` of `redirect_output`.

```suggestion
from typing import Any, Callable, Container, Optional, Union
```